apps/etl-func/src/adapters/pdv-tokenizer/token-resource.ts

```diff
@@ -0,0 +1,6 @@
+import { z } from "zod";
+
+export const tokenResourceSchema = z.object({
+  token: z.string().uuid().min(1),
+});
```

Comment:
Ditto, this could be just a string, also I think  we do not need to specify `min(1)` if this is an UUID
```suggestion
export const tokenResourceSchema = z.string().uuid();
```

Reply:
Ditto. This is the response of the pdv tokenizer api and it is an object that contains a field name token that is a uuid. I agree that we can remove the min(1) thought